### ISSUE TITLE
TSV export function generates a CSV file

### ISSUE THREAD

**Issue by tedyapo** [markummitchell/engauge-digitizer#169]
I tried exporting the same data to TSV and CSV files, and the resulting export files are identical - both CSV files.

The version is 9_1 AppImage for Linux64

Awesome software, BTW!  Thank you!

EDIT: just saw this as a dup of #53

**Comment by markummitchell** [comment]
You're welcome. Always good to see people using the software.

In the Export Format dialog, there is checkbox for 'Override in CSV/TSV files'. That should not be checked if you want the "standard behavior", which is that CSV files get comma delimiters (consistent with Comma-Separated Values) and TSV files get tab delimiters (consistent with Tab-Separated Values), and all other files get the delimiter specified in the Export Format dialog.

**Comment by markummitchell** [comment]
Is the code working right for you?

**Comment by tedyapo** [comment]
I got it to export with the desired delimiters, but in the Appimage (linux
64-bit), the "override" is checked by default - I just checked an a fresh
machine. So if you:

1.) Run the Appimage for the first time
2). import png, choose 3 axis points, choose a number of data points
3) export as csv - get commas
4) export as tsv - get commas

If you then uncheck the "override" box as you mentioned, you get the
behavior expected by a naive user. I was exporting data within a few
minutes of first using the program, and didn't even know the "Export
Format" menu existed when I first saw this behavior. Maybe the box should
be unchecked by default; if the user is not satisfied with the default
csv/tsv behavior, they can dig deeper, find the menu, and make changes
(comprehensive set of options there, BTW, I got exactly what I needed to
export into octave and gnuplot).  As it stands, it's easy to think there's
a bug.
